### ISSUE TITLE
Section 'oiledmachine' in repos.conf has name different from repository name 'oiledmachine-overlay' set inside repository

### ISSUE THREAD

**Issue by meng89** [orsonteodoro/oiledmachine-overlay#1]
When I finished layman -a oiledmachine, I get the message as this issue title, and none of any packages in this overlay can be search or install by emerge.

**Comment by orsonteodoro** [comment]
Try renaming [oiledmachine] to [oiledmachine-overlay] in layman.conf or one of the *.conf files in /etc/portage/repos.conf:

#change location to the location of the folder
#location = /usr/local/portage
contents of /etc/portage/repos.conf/oiledmachine-overlay.conf
[oiledmachine-overlay]
location = /usr/local/portage
priority = 100
